### ISSUE TITLE
Docker: Permission Denied Errors after 3.5.1

### ISSUE THREAD

**Issue by dubhunter** [sabnzbd/sabnzbd#2105]
I updated to 3.5.1 the other day, and since then I am getting the following error:

```Failed making (/data/downloads/THING
Traceback (most recent call last):
  File "/app/sabnzbd/sabnzbd/filesystem.py", line 719, in create_all_dirs
    os.mkdir(path_part_combined)
PermissionError: [Errno 13] Permission denied: '/data/downloads/THING'
```

**Comment by dubhunter** [comment]
I cranked it all the way up to 777 and it is still not working.

The weird thing is it always worked prior to this version. When I look at the files (from previous versions) they are all owned by 911:911. I am running via docker on Synology.

**Comment by dubhunter** [comment]
I just tried booting the container with a user known to be able to write to the folder, and now I am getting more errors:
```
Failure in tempfile.mkstemp
Error importing THING
...```

**Comment by dubhunter** [comment]
I was able to fix things.

I started the containers with a known user/group that has access to the folders. Then I needed to make sure the Sab download folders were mapped to the correct folder (at some point they went to `/downloads` instead of `/data/downloads`)

Sooooooo, there was some user error as well as a change in Sab behavior. For anyone else using Docker/Synology that runs into this...
Find the uid/gid for a user (in my case `docker`) that has access to the mounted volume.
Then add `PUID` and `PGID` to the Environment section of the container config.
This will cause the container to run as that user and then will obey the permissions you set in Synology.
